src/Accelerators/NNPA/Conversion/ONNXToZHigh/RewriteONNXForZHigh.cpp

```diff
@@ -183,13 +189,84 @@ bool isDefinedByONNXConstantOp(Value v) {
   return isa<ONNXConstantOp>(v.getDefiningOp());
 }
 
+bool LegalExpandPowOpToMul(ONNXPowOp op) {
```

Comment:
Legal or Illegal? The return value seems to mean illegal to me.

Reply:
It's legal to the conversion target. Anyway, I change its name to avoid confusion. Hope it's easy to interpret now. Thanks!